### ISSUE TITLE
[Feature]: Additional domain

### ISSUE THREAD

**Issue by mhackdo18** [t0bst4r/matterbridge-home-assistant-addon#78]
### Domain

Button, Climate

### Attributes

```shell
Please add button and climate domain. thanks
```

### Version

1.3.5-68

**Comment by t0bst4r** [comment]
Climates were already requested in the main project. t0bst4r/matterbridge-home-assistant#47

What exactly do you need with buttons? You have a button connected to HA, and when pressed you want that event to be pushed through HA to matter and then to the connected controller?
E.g. to trigger an Alexa Routine with a button which would not be supported with Alexa otherwise?

**Comment by mhackdo18** [comment]
Hi,

Yes. Mostly buttons works like scene on google home. There are entities
with button press to trigger.

**Comment by MarcoSchicke** [comment]
Yes, this would be great, to trigger other devices e.g. from Alexa via Home Assistant.

**Comment by jvmahon** [comment]
I think you're talking about adding support for the Matter Generic Switch device class (which, despite being called a switch, is really used for button press events). I don't believe the Matter Generic Switch is supported by Alexa (or Google Home) though it can be supported in SmartThings (I think you need to do a custom driver though). I think the only controller that currently supports Generic Switch for button tap detection is iOS.
